Dockerfile

```diff
@@ -39,3 +39,6 @@ COPY . $DIR
 RUN cd bashbrew/go && gb build
 
 VOLUME $BASHBREW_CACHE
+
+COPY docker-entrypoint.sh /docker-entrypoint.sh
```

Comment:
This is already copied into `$DIR` above, so this can be a symlink instead, and I'd rather we put this into the repo as `bashbrew/entrypoint.sh` and into the `PATH` as `bashbrew-entrypoint.sh` (so that our base image's `entrypoint` is still available via `docker-entrypoint.sh`; https://github.com/docker-library/docker/blob/99287145029122cf49321fa2a055d14240001d1d/1.11/Dockerfile#L21-L24)

ie:
```dockerfile
RUN ln -s "$PWD/bashbrew/entrypoint.sh" /usr/local/bin/bashbrew-entrypoint.sh
ENTRYPOINT ["bashbrew-entrypoint.sh"]
```